attelo/decoding/control.py

```diff
@@ -60,10 +60,14 @@ def _predict_relate(dpack, models):
     and an list of labels
     """
     if models.relate == 'oracle':
-        idxes = dpack.target
+        unrelated = dpack.label_number(UNRELATED)
```

Comment:
We should probably add `unk_lbl = 0`, or even better, `unk_lbl = dpack.label_number(UNKNOWN)` with the unknown label automatically added to the label encoder, as it is done in `educe`.
Then the anonymous function below should be adapted: ...`lambda lbl: unk_lbl if lbl == unrelated else x`.